Scripts/build.swift

```diff
@@ -23,49 +23,61 @@ enum TaskError: Error {
 enum Platform: String, CaseIterable, CustomStringConvertible {
     case iOS_12
     case iOS_13
+    case iOS_14
     case tvOS_12
     case tvOS_13
+    case tvOS_14
     case macOS_10_15
     case watchOS_5
     case watchOS_6
+    case watchOS_7
 
     var destination: String {
         switch self {
         case .iOS_12:
-            return "platform=iOS Simulator,OS=12.2,name=iPad Pro (12.9-inch) (3rd generation)"
+            return "platform=iOS Simulator,OS=12.4,name=iPad Pro (12.9-inch) (3rd generation)"
```

Comment:
GitHub actions has different simulators pre-installed than Travis did, so we update them as part of this PR